src/tests/ftest/daos_test/suite.py

```diff
@@ -443,3 +443,18 @@ def test_daos_upgrade(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_upgrade
         """
         self.run_subtest()
+
+    def test_daos_pipeline(self):
+        """Jira ID: DAOS-13160
+
+        Test Description:
+            Run daos_test -P
+
+        Use cases:
+            Core tests for daos_test
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium,provider
+        :avocado: tags=daos_test,daos_core_test,test_daos_pipeline
```

Comment:
If you need to push again...
```suggestion
        :avocado: tags=all,pr,daily_regression
        :avocado: tags=hw,medium,provider
        :avocado: tags=daos_test,daos_core_test
        :avocado: tags=DaosCoreTest,test_daos_pipeline
```

Reply:
i guess i copied/pasted the wrong test (the one before it)..